### ISSUE TITLE
Check red/green/yellow colors for accessibility

### ISSUE THREAD

**Issue by dboyer** [WikiWatershed/mmw-micro#18]
The red, yellow, and green colors on the water column may fail accessibility checks for color blindness. 

Learning specialists recommended using https://developer.paciellogroup.com/resources/contrastanalyser/  to check color contrast.

Note: changing these colors may impact colors used in other parts of MMW.

![Screen Shot 2020-05-26 at 1 58 16 PM](https://user-images.githubusercontent.com/516764/83174741-cff4b880-a0e8-11ea-8432-6135b75c3246.png)

**Comment by dboyer** [comment]
@designmatty Could use a design consult on this.

**Comment by designmatty** [comment]
I ran the app with **Protanopia** color blindness simulation and the results are pretty telling of much needed change.

**CURRENT COLORS: No vision deficiency:**
![image](https://user-images.githubusercontent.com/1928955/86172425-50de1000-baec-11ea-9ff5-bdb9047a3795.png)

**CURRENT COLORS: Protanopia:**
![image](https://user-images.githubusercontent.com/1928955/86167773-2177d500-bae5-11ea-91e6-cca8a9977bfd.png)

All of the colors are shades of orange/brown. It becomes difficult to tell which arrow relates to which attribute. The arrows blend into the illustration.

---

The following adjustments would likely get us to a pretty decent fix.

**NEW COLORS: No vision deficiency:**
![image](https://user-images.githubusercontent.com/1928955/86172264-17a5a000-baec-11ea-8935-16a19d1114a6.png)

**NEW COLORS: Protanopia**
![image](https://user-images.githubusercontent.com/1928955/86172331-2db36080-baec-11ea-946a-0461a51a084f.png)

The new colors are all visually distinct. There were still some issues with the arrows blending into the illustration so I added an outline to help distinguish the shapes.

**Comment by rajadain** [comment]
Thanks for putting this together! Should we also check for Deuteranopia?

Also, just to note, that these color changes will be carried through in the TR-55 model in the main app as well:

![image](https://user-images.githubusercontent.com/1430060/86387269-f1554100-bc60-11ea-8d50-4327498b7b3e.png)

**Comment by designmatty** [comment]
**This is how it might look for Deuteranopia:** It's fairly similar to Protanopia

![image](https://user-images.githubusercontent.com/1928955/86398852-43ec2880-bc74-11ea-91f7-1b971c911055.png)

**Comment by rajadain** [comment]
Thanks for confirming, that looks good to me

**Comment by dboyer** [comment]
@designmatty We've received a request to also check for tritanopia (blues). Could you see how they look with that change?

**Comment by rajadain** [comment]
From this online simulator: https://www.color-blindness.com/coblis-color-blindness-simulator/

I'm seeing this for tritanopia:

![image](https://user-images.githubusercontent.com/1430060/86964378-f835f500-c133-11ea-984b-465ca548efa3.png)

In fact, the only case when I see problems if for monochromacy (or achromatopsia), those who can't see any color:

![image](https://user-images.githubusercontent.com/1430060/86964605-5a8ef580-c134-11ea-8fdb-49b95ed758bc.png)

which is pretty rare (~1 / 30K people have it). I don't know if we are under obligation to support that.

If we do, we may have to move to using patterns in addition to colors, but that would be a larger design effort.

**Comment by rajadain** [comment]
@designmatty Can you post the hexcodes for the new colors please?

**Comment by designmatty** [comment]
@rajadain:

Evapotranspiration: #2d8bff
Infiltration: #f8aa00
Runoff: #ed3b3b

**Comment by rajadain** [comment]
@designmatty the infiltration color listed above seems to be the same as before, more the orange than the yellow in your screenshot
